### PR TITLE
reset lease time after bootstrap

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: meta/meta.go [hunk 1]
```diff
@@ -517,3 +518,19 @@ func (m *Meta) GetHistoryDDLJob(id int64) (*model.Job, error) {
 	err = job.Decode(value)
 	return job, errors.Trace(err)
 }
+
+// IsBootstrapped returns whether we have already run bootstrap or not.
+// return true means we don't need doing any other bootstrap.
+func (m *Meta) IsBootstrapped() (bool, error) {
+	value, err := m.txn.GetInt64(mBootstrapKey)
```
Comment (qiuyesuifeng) [inline meta/meta.go]: Why not use Get but GetInt64， seems in line 534 u set "1".

File: domain/domain.go [hunk 2]
```diff
@@ -107,12 +115,14 @@ func (do *Domain) reload() error {
 	return errors.Trace(err)
 }
 
-func (do *Domain) loadSchemaInLoop() {
-	if do.lease <= 0 {
-		return
+func (do *Domain) loadSchemaInLoop(lease time.Duration) {
+	// check schema every 300 seconds default.
+	const defaultLoadTime = 300 * time.Second
```
Comment (qiuyesuifeng) [inline domain/domain.go]: How about defining this variable out of this function?

File: meta/meta.go (hunk 1 above)
Comment (siddontang) [inline meta/meta.go]: For set "1", we can use GetInt64 to fetch this number directly, no need to do another conversation.

**Pull request merged**